Services/CO.CDP.Organisation.WebApi.Tests/UseCase/UpdateOrganisationUseCaseTest.cs

```diff
@@ -202,6 +202,39 @@ public async Task Execute_ShouldPublishOrganisationUpdatedEvent_WhenOrganisation
 
     }
 
+    [Fact]
+    public async Task Execute_ShouldNotInsertIdentifier_WhenIdentifierIdIsEmptyOrNull()
+    {
+        var command = new UpdateOrganisation
+        {
+            Type = OrganisationUpdateType.AdditionalIdentifiers,
+            Organisation = new OrganisationInfo
+            {
+                AdditionalIdentifiers = [new OrganisationIdentifier
+                {
+                    Id = "",
+                    LegalName = "Acme",
+                    Scheme = "GB-VAT"
+                },
+                new OrganisationIdentifier
+                {
+                    Id = null,
+                    LegalName = "Acme",
+                    Scheme = "GB-VAT"
+                }]
+            }
+        };
+        var organisation = Organisation;
+        _organisationRepositoryMock.Setup(repo => repo.Find(_organisationId)).ReturnsAsync(organisation);
+
+        var result = await UseCase.Execute((_organisationId, command));
+
+        result.Should().BeTrue();
+        _organisationRepositoryMock.Verify(repo => repo.Save(organisation!), Times.Once);
+
+        organisation.Identifiers.FirstOrDefault(i => i.Scheme == "GB-VAT").Should().BeNull();
+    }
+
```

Comment:
This only handles the case of adding an identifier. What about the case when someone adds VAT but then changes their answer?

Reply:
I mean the situation when someone gives the VAT number at first:
<img width="780" alt="image" src="https://github.com/user-attachments/assets/f21c11a0-e5db-4a9d-80e0-fb1c8ae09277">

But then, they change their answer to "No, I don't have a VAT number":

<img width="872" alt="image" src="https://github.com/user-attachments/assets/e3313718-2284-40b9-bc17-8727fbcb5ba2">

Reply:
Added Execute_ShouldUpdateVatNumber_WhenVatIdentifierAlreadyExists